modules/mysql/charts.go

```diff
@@ -462,6 +462,34 @@ var charts = Charts{
 			{ID: "open_tables", Name: "tables"},
 		},
 	},
+	{
+		ID:    "process_list_fetch_duration",
+		Title: "Process List Fetch Duration",
+		Units: "milliseconds",
+		Fam:   "process list",
+		Ctx:   "mysql.process_fetch_query_duration",
+		Dims:  Dims{{ID: "process_fetch_query_duration", Name: "Query Duration"}},
```

Comment:
```suggestion
		Ctx:   "mysql.process_list_fetch_query_duration",
		Dims:  Dims{{ID: "process_list_fetch_query_duration", Name: "duration"}},
```

⚠️ don't submit the changes via GitHub

- let's add **_list** (other metrics have it)
- lowercase dimension name. Also, i think just "duration" is ok